src/System.Net.Sockets/src/System/Net/Sockets/SocketPal.Unix.cs

```diff
@@ -694,7 +685,7 @@ public static unsafe bool TryCompleteReceiveMessageFrom(SafeCloseSocket socket,
             }
         }
 
-        public static bool TryCompleteSendTo(SafeCloseSocket socket, byte[] buffer, ref int offset, ref int count, SocketFlags flags, byte[] socketAddress, int socketAddressLen, ref int bytesSent, out SocketError errorCode)
+        public static bool TryCompleteSendTo(SafeCloseSocket socket, Span<byte> buffer, ref int offset, ref int count, SocketFlags flags, byte[] socketAddress, int socketAddressLen, ref int bytesSent, out SocketError errorCode)
```

Comment:
I think we could simplify this signature somewhat.  We really just need to have an out param for bytesSent, I think.  That may be more trouble than it's worth here, though.

Reply:
Ok.  If you don't mind, let's hold off on that as part of this PR. It can be revisited subsequently.